### PR TITLE
Fix jsdoc warnings

### PR DESCRIPTION
**Pull request by raimund-schluessler**
This fixes the jsdoc warnings except the jsdoc/no-undefined-types warnings.

@skjnldsv Do you know what the proper way of fixing the no-undefined-types warnings is?

### CODE REVIEW

File: src/fullcalendar/eventSources/freeBusyBlockedForAllEventSource.js [hunk 1]
```diff
@@ -153,8 +151,8 @@ export default function(organizer, attendees, resources) {
 }
 
 /**
- * @param slots
- * @param start
+ * @param {object} slots the slots
+ * @param {object} start the start
```
Comment (tcitworld) [inline src/fullcalendar/eventSources/freeBusyBlockedForAllEventSource.js:155]: `start` does not seem to be an object.

Comment (raimund-schluessler) [reply src/fullcalendar/eventSources/freeBusyBlockedForAllEventSource.js:155]: Wasn't sure. Is it a string?

Comment (tcitworld) [reply src/fullcalendar/eventSources/freeBusyBlockedForAllEventSource.js:155]: I checked, it should be a simple `Date` object (comes from ICAL.js via calendar-js).

Comment (raimund-schluessler) [reply src/fullcalendar/eventSources/freeBusyBlockedForAllEventSource.js]: Fixed.

**Comment by st3iny** [comment]
> Do you know what the proper way of fixing the no-undefined-types warnings is?

Importing them should work but there are a lot of fictional types which we can just ignore. I'd be fine with merging now because most of the warnings are already gone.

**Review by st3iny — APPROVED** [review]

**Comment by raimund-schluessler** [comment]
> 
> 
> > Do you know what the proper way of fixing the no-undefined-types warnings is?
> 
> Importing them should work but there are a lot of fictional types which we can just ignore. I'd be fine with merging now because most of the warnings are already gone.

I would anyway fix that in a follow-up PR. So merging is fine with me. 👍

**Review by tcitworld — APPROVED** [review]

**Pull request merged**